R/arith.R

```diff
@@ -14,6 +14,8 @@
 #' @param op An arithmetic operator as a string
 #' @param x,y A pair of vectors. For `!`, unary `+` and unary `-`, `y` will be
 #'   a sentinel object of class `MISSING`, as created by `MISSING()`.
+#' @inheritParams ellipsis::dots_empty
```

Comment:
Does this do anything? The argument of `check_dots_empty()` is `env`?

Reply:
This inherits from `dots_empty` which was written for this purpose: https://github.com/r-lib/ellipsis/blob/d8bf8a34eb3be04a52169bc69759ebccc7f247e2/R/dots.R#L6

Reply:
Ah got it.